assets/js/agent/app.vue

```diff
@@ -64,6 +81,17 @@
         var $data = this.$data;
         var me = this;
 
+        loadAllChannels(this.realmq, function (err, channels) {
+          if (err) {
+            $data.channels = [];
+            return;
+          }
+          $data.channels = channels.sort(function (a, b) {
+            return a.createdAt < b.createdAt ? 1 : -1;
+          }).map(me.getChannelViewModel);
+          me.activateChannel(channels.length && channels[0]);
+        });
+
         this.realmq.channels.list().then(function (channelList) {
```

Comment:
shouldn't we remove the old implementation?

Reply:
Yes, done it.